src/antsibull/data/sphinx_init/conf_py.j2

```diff
@@ -28,7 +28,8 @@ intersphinx_mapping = {
     'python': ('https://docs.python.org/2/', (None, '../python2.inv')),
     'python3': ('https://docs.python.org/3/', (None, '../python3.inv')),
     'jinja2': ('http://jinja.palletsprojects.com/', (None, '../jinja2.inv')),
-    'ansible4': ('https://docs.ansible.com/ansible/4/', (None, '../ansible4.inv')),
+    'ansible_devel': ('https://docs.ansible.com/ansible/devel/', (None, '../ansible_devel.inv')),
+    'ansible5': ('https://docs.ansible.com/ansible/5/', (None, '../ansible5.inv')),
```

Comment:
```suggestion
    # If you want references to resolve to a released Ansible version (say, `5`), uncomment and replace X by this version:
    # 'ansibleX': ('https://docs.ansible.com/ansible/X/', (None, '../ansibleX.inv')),
```

Reply:
This makes sense to me, but any reason you feel it's better to use a specific version rather than `devel`? 
Or, I just realized that `latest` seems to work (at least in the URLs themselves), so if we don't want `devel`, maybe we can sidestep the need to update the versions by using `latest` instead?